tests/bootstrap_project/test/project_test.dart

```diff
@@ -0,0 +1,432 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:serverpod/serverpod.dart';
+import 'package:test/test.dart';
+import 'package:path/path.dart' as path;
+
+void main() async {
+  Directory.current = path.join(Directory.current.path, '..', '..');
+  final rootPath = Directory.current.path;
+  final cliPath = path.join(rootPath, 'tools', 'serverpod_cli');
+  final tempPath = path.join(rootPath, 'temp');
+
+  final timeout = Timeout(Duration(minutes: 5));
+
+  setUpAll(() async {
+    await Process.run(
+      'dart',
+      ['pub', 'global', 'activate', '-s', 'path', '.'],
+      workingDirectory: cliPath,
+    );
```

Comment:
Thought (Does not require action) - This will change the path for serverpod for the user of the host, having unexpected behavior if run locally.

I wonder if we could spawn a single shell process that we interact with to run commands. In case we could get away with only locally changing the path to the CLI.

Not sure how this would work with awaiting command executions and getting the exit code of the previous executed process and such.

Reply:
Yeah this is a drawback with this approach, I think your solution might be impossible to do inside the test, not sure. But you could make an argument that the test shouldn't manipulate the path at all. 

But then we would have to add this in the CI script and you would have to manually set this if running the test locally. However, that might be a good thing it means you can test whatever CI command you want. 

I'm not set on what is the best solution.